reactor-core/src/main/java/reactor/core/publisher/MonoPeekFuseable.java

```diff
@@ -105,7 +102,7 @@ public Runnable onCompleteCall() {
 	@Override
 	@Nullable
 	public Runnable onAfterTerminateCall() {
-		return onAfterTerminateCall;
```

Comment:
maybe mark this method as "unused" in the javadoc to remove any ambiguity (`FluxPeekFuseable` uses it, `MonoPeekFuseable` doesn't and favors `MonoPeekTerminal` instead)